Documentation/Cluster.rst

```diff
@@ -83,22 +87,24 @@ Get Cluster information
 Off-line migration
 ------------------
 
-* VMs can be migrated to the other node in the cluster, whilst the VM is powered off, using::
+* DRBD VMs can be migrated to the other node in the cluster, whilst the VM is powered off, using::

```

Comment:
For consistency, could you say something like 'VMs that use DRBD-based storage'? :)